.gitignore

```diff
@@ -40,7 +40,6 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
-package-lock.json
```

Comment:
Hey, @amaredeus, don't we want this to be ignored, in the case that someone uses npm to install dependencies and run scripts?

Reply:
This came up during a troubleshooting session with @geositta. The concern was that there should only be 1 lock file for the build out, either yarn or npm, if both are included that might cause a security vulnerability. In the case of birdman, the npm lock file was present and completely iginored by git. Even though he reverted all changes using git, the npm lock file was still present. Removing the file from .gitignore will make sure git sees the file and we can reject it if it gets included in a future PR

Reply:
Gotcha. Okay. Thank you for clarifying this.